homeassistant/components/cover/tahoma.py

```diff
@@ -0,0 +1,92 @@
+"""
+Support for Tahoma cover - shutters etc.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.tahoma/
+"""
+import logging
+
+from homeassistant.components.cover import CoverDevice, ENTITY_ID_FORMAT
+from homeassistant.components.tahoma import (TAHOMA_DEVICES, TahomaDevice)
+
+REQUIREMENTS = ['tahoma-api==0.0.6']
```

Comment:
The component need to take care about the requirements. Remove it in the platform.